ProxyAppsSuite/tests/AMG.yaml

```diff
@@ -1,92 +1,58 @@
-_base:
-
-    summary: A test of amg
-
+amg:
+    summary: P1. Laplace problem on 3D cube with 27 point stencil. P2. Non-linear time-dependent problem.
+    variables:
+        problem: 
+            - { name: "problem1", nxval: '100', nyval: '100', nzval: '100', pxval: '2', pyval: '2', pzval: '1', flags: '-problem 1' }
+            - { name: "problem2", nxval: '20', nyval: '20', nzval: '20', pxval: '2', pyval: '2', pzval: '1', flags: '-problem 2' }
+    slurm:
+        num_nodes: 4
```

Comment:
Did you intend to set the number of nodes to 4?  The way this runs, now, it's running a 4 node job, 1 MPI rank per node, two threads per rank.  I suggest the following:
```yaml
    slurm:
        num_nodes: 1
        tasks_per_node: 12
```
        num_nodes: 1
        tasks_per_node: 12. # max on Woodchuck
```
and including the adjustments to the problem variables above.